File: AssignifyIt.Models/DailyText.cs
using System;

namespace AssignifyIt.Models
{
    public class DailyText
    {
        public Guid Id { get; set; } 
        public string DateLine { get; set; } 
        public string Header { get; set; } 
        public string Body { get; set; }
        public DateTime DateEntered { get; set; } 
    }
}

File: DailyText/daily-text.js
$(function () {
    //$('#header-text').text('Daily Text');

    $.getJSON("http://assignit.apphb.com/DailyText/Json", null, function (result) {
        var textDate = $('#text-date');
        var textheader = $('#text-header');
        var textBody = $('#text-body');

        textDate.append(result.DateLine);
        textheader.append("<i>" + result.Header + "</i>");
        textBody.append(result.Body);

    });
});